<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CyberCalc52</class>
 <widget class="QDialog" name="CyberCalc52">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>298</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>CyberCalc52</string>
  </property>
  <widget class="QFrame" name="BaseButtonFrame">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Button" name="btn_B_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_11">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_12">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_13">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_14">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_15">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_16">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_17">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_18">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_19">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_B_20">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="AddButtonFrame">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Button" name="btn_A_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_11">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="Button" name="btn_A_12">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>190</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="ICDFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QFrame" name="SliderFrame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>471</width>
      <height>111</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="Switch" name="checkBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
    <widget class="Switch" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Switch" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>40</y>
       <width>90</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Switch" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>90</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>CheckBox</string>
     </property>
    </widget>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>471</width>
      <height>151</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>11</number>
    </property>
    <property name="value" stdset="0">
     <double>0.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Switch</class>
   <extends>QCheckBox</extends>
   <header>switch.h</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
